### PR TITLE
Helm wip

### PR DESCRIPTION
**Pull request by Sedward**
Here is one more pull request to get the persistent volume claim working and some aesthetic changes based on some issues I ran into when naming specs. Again, maybe not everything gets merged. If there are any questions, let me know. 

At this point we should consider using github issues to identify what changes to make and assign them to contributors. That way we aren't stepping on each others toes with commit. In addition, we can discuss certain changes before actually making them and ensure everyone agrees on the proposed changes.  I'd even go as far as saying we could maybe move this the charts/incubator or it's own temporary repo.

### CODE REVIEW

**Comment by gsemet** [comment]
Can you rebase on my branch? I did a few hard push.

and I agree with tickets, please open them !

File: airflow/templates/configmaps.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: "{{ .Values.airflow.prefix -}}airflow-cfg"
+  name: {{ template "airflow.fullname" . }}-airflow-cfg
```
Comment (gsemet) [inline airflow/templates/configmaps.yaml:4]: I don't exactly see the point?

Don't you want something like
```
{{ template "airflow.fullname" airflow-cfg }}
```

File: airflow/templates/services.yaml [hunk 2]
```diff
@@ -1,67 +1,67 @@
...
-  name: {{ .Values.airflow.prefix -}}rabbitmq
+  name: {{ template "airflow.fullname" . }}-rabbitmq
 spec:
   type: ClusterIP
   selector:
     app: airflow
     tier: rabbitmq
   ports:
-    - name: {{ .Values.airflow.prefix -}}node
+    - name: rabbitmq
```
Comment (gsemet) [inline airflow/templates/services.yaml:26]: please keep the prefix, I need it to allow several airflow in the same namespace

File: airflow/values.yaml [hunk 3]
```diff
@@ -43,14 +40,16 @@ db:
...
+  enabled: true
   ## storageClass: Persistent Volume Storage Class
   ## If defined, storageClassName: <storageClass>
   ## If set to "-", storageClassName: "", which disables dynamic provisioning
   ## If undefined (the default) or set to null, no storageClassName spec is
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   # storageClass: default
+  # If defined, will overide the claim name
+  claimName: "killjoy-dragonfly"
```
Comment (gsemet) [inline airflow/values.yaml]: should be in your config.yaml

File: airflow/values.yaml [hunk 4]
```diff
@@ -43,14 +40,16 @@ db:
     database: airflow
 
 persistence:
-  enabled: false
+  enabled: true
```
Comment (gsemet) [inline airflow/values.yaml]: set to false by default, user will enable it in their config.yaml

File: reset-af.sh [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (gsemet) [inline reset-af.sh]: I prefere using the makefile. works great

**Review by gsemet — CHANGES_REQUESTED** [review]

**Comment by Sedward** [comment]
Making fixes. I pushed too hastily.

**Comment by gsemet** [comment]
Sorry I pushed other commits, please rebase

File: airflow/templates/configmaps.yaml (hunk 1 above)
Comment (Sedward) [reply airflow/templates/configmaps.yaml:4]: In the docs they have something like this

apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ .Release.Name }}-configmap

I was mostly trying  to conform to this. I haven't tried this syntax but I can.

File: airflow/templates/services.yaml (hunk 2 above)
Comment (Sedward) [reply airflow/templates/services.yaml:26]: I understand. The issue I ran into what the Release Name was somewhat long and combined with the service, it surpassed the 15 character limit.

File: airflow/values.yaml (hunk 4 above)
Comment (Sedward) [reply airflow/values.yaml]: agreed. that was a mistake

File: airflow/templates/configmaps.yaml (hunk 1 above)
Comment (gsemet) [reply airflow/templates/configmaps.yaml:4]: ok i'll see what this do tomorow. My requirement is to be able to change this release easily so we can have several instance of airflow in the same namespace (especially, we share the namespace with another celery+rabbitmq), but we are also trying kubernetes executor fork, all in the same namespace

File: airflow/templates/services.yaml (hunk 2 above)
Comment (gsemet) [reply airflow/templates/services.yaml:26]: yes it needs to be small and dns compatible

**Comment by Sedward** [comment]
I think we need to enable issues for this repo? I don't see the issue at the top.

**Comment by gsemet** [comment]
Done !

**Pull request closed without merging**